### PR TITLE
Add Kafka docs

### PR DESCRIPTION
**Pull request by laurensslats**

### CODE REVIEW

File: docs/Cloud_integrations/images/kafka-configuration.png
Comment (mikegpl) [inline docs/Cloud_integrations/images/kafka-configuration.png:1]: I think we should at least blur here the parts of `bootstrap.servers` and `sasl.jaas.config` where the credentials are visible

File: docs/Cloud_integrations/images/kafka-config-set.png
Comment (mikegpl) [inline docs/Cloud_integrations/images/kafka-config-set.png:1]: Mind that `domainId` is here set to. `/lwm2m_workshops/grabowski` and in other places it is `/embedded`

File: docs/Cloud_integrations/images/kafka-event-arrived.png
Comment (mikegpl) [inline docs/Cloud_integrations/images/kafka-event-arrived.png:1]: The same goes for domain here

File: docs/Cloud_integrations/kafka.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+title: Kafka
+date: 2023-08-08
+og_title: AVSystem IoT Developer Zone
+---
+
+# Kafka
+
+Connect {{ coiote_long_name }} to your Kafka cluster by creating a Kafka event handler.
```
Comment (mikegpl) [inline docs/Cloud_integrations/kafka.md:9]: I think it's worth pointing out that you can connect with `Azure Event Hub`, e.g. by writing `to your Kafka cluster or Azure Event Hub` - I think more people are used to working with Azure services than to working with Kafka clusters (either managed by themselves or by other companies)

File: docs/Cloud_integrations/kafka.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    - Device successfully executes a firmware update
+    - Device deleted
+
+    **Telemetry events** include data which is generated by the device. Events can originate from Objects, Object Instance, Resources or Resource Instances.
+
+![Select event triggers](images/events.png "Select event triggers")
+
+## Configure Kafka parameters
+
+Specify the **Topic** and **Producer configuration** to set up the conntion with a Kafka cluster.
```
Comment (mikegpl) [inline docs/Cloud_integrations/kafka.md:50]: Typo: `conntion` => `connection`

**Review by mikegpl — CHANGES_REQUESTED** [review]

File: docs/Cloud_integrations/images/kafka-config-set.png
Comment (laurensslats) [reply docs/Cloud_integrations/images/kafka-config-set.png:1]: Good observation, this doesn't hurt IMHO

**Review by mikegpl — APPROVED** [review]
LGTM :)

**Pull request merged**